### DISCUSSION TITLE
Are there alternatives to the list (global list of mirror, scraping websites)?

### DISCUSSION THREAD

**Discussion by ghost** [quenhus/uBlock-Origin-dev-filter#389] [General]
Specifically, I'm after a filter list cataloging all websites which exclusively scrape content from other websites. Preferably has another version for DNS use. [This project's SEO spam list](https://github.com/quenhus/uBlock-Origin-dev-filter/blob/main/data/seo_spam.txt) is under 50 lines in length and the readme implies that all domains may not be added:

> This uBlock filter is my own filter, for my usage and can't obviously satisfy everyone.

**Comment by quenhus** [comment]
Hi, thank you for your interest in this project.

Indeed, I'm going to keep the "dev scope" for this project. However, I am open to contribution if you have a list of bad domains with justification for each one.

I don't know any uBlock/uBlacklist list which filters out domains that clone others.

You can use my list with uBlacklist and add others domains. You might also use [letsblock.it](https://letsblock.it/filters/search-results) to do so with uBlock.
